off-chain/src/Network/Types.purs

```diff
@@ -130,15 +130,16 @@ derive instance Newtype GetActionStatus _
 
 data ActionStatus
   = AskForSignature
-      { uuid :: UUID
+      { uuid :: UUID -- Remove it? we don't need anymore with current types.
```

Comment:
Hmm. The one way I can see it could be useful, is that user can ignore `AskForSignature` responses that don't have known UUIDs.

Reply:
I can try to remove it before merge.